tests/angular_devkit/build_webpack/angular-app/tsconfig.json

```diff
@@ -8,7 +8,7 @@
     "moduleResolution": "node",
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
-    "target": "es5",
+    "target": "es2015",
```

Comment:
This is to avoid having NGCC go through and transform ES5 entrypoints, currently NGCC is taking more time than expected to process `@angular/core`.